lib/horde/dynamic_supervisor_impl.ex

```diff
@@ -649,6 +646,49 @@ defmodule Horde.DynamicSupervisorImpl do
     |> Map.put(:local_process_count, new_local_process_count)
   end
 
+  defp handoff_child(child, state) do
+    reply =
+      Horde.ProcessesSupervisor.terminate_child_by_id(
+        supervisor_name(state.name),
+        child.id,
+        :horde_handoff
```

Comment:
I've had a think about the proper exit code, and I think it should be `{:shutdown, :process_redistribution}`

Reply:
I'm cool with it 👍 - will include this in next commit

Reply:
Resolved in [692028a](https://github.com/derekkraan/horde/pull/164/commits/692028ab2aed36e5f36148b7ca29be21ff6de286)